README.md

```
@@ -37,3 +37,14 @@ This is a MERN stack
 - Card Form is not more user friendly
 - Disabled IA generation for new questions for the moment
   > TODO : Adjust card creation option
+
+### 20/10/2025
+
+- Added Husky
+- Eslint additionnal rules
+
+#### Install Husky on machine
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)